polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -2278,61 +2278,63 @@ MLIRASTConsumer::GetOrCreateLLVMGlobal(const ValueDecl *FD,
 }
 
 std::pair<mlir::memref::GlobalOp, bool>
-MLIRASTConsumer::GetOrCreateGlobal(const ValueDecl *FD, std::string prefix,
-                                   bool tryInit, FunctionContext funcContext) {
-  std::string name = prefix + CGM.getMangledName(FD).str();
-
-  name = (PrefixABI + name);
+MLIRASTConsumer::getOrCreateGlobal(const ValueDecl &VD, std::string Prefix,
```

Comment:
Notes:
Commit 8f27d089121cad42fffbb1d795f16b89946bc2eb is NFC.
The remaining 2 commits in this PR address respectively:
  - setting the global "constant"
  - setting the alignment attribute on the global